docs/simplesamlphp-customauth.md

```diff
@@ -272,6 +273,12 @@ The class follows:
                 throw new Exception('Missing or invalid password option in config.');
             }
             $this->password = $config['password'];
+            if (isset($config['options']) {
+                if (!is_array($config['options])) {
+                    throw new Exception('Missing or invalid options option in config.');
+                }
+                $this->options = $config['options'];
+            }
         }
```

Comment:
A bit of a nitpick because it's documentation, but $this->options will now be unset if you don't have the config option, thus generate a notice below when it's used.